.github/workflows/pipeline.yml

```diff
@@ -340,6 +340,7 @@ jobs:
           echo "LANGFUSE_ASYNC_INGESTION_PROCESSING=true" >> .env
           echo "LANGFUSE_CACHE_API_KEY_ENABLED=true" >> .env
           echo "LANGFUSE_CACHE_PROMPT_ENABLED=true" >> .env
+          echo "CLICKHOUSE_CLUSTER_ENABLED=false" >> .env
```

Comment:
This shouldn't be necessary as we already cover it via `cp .env.dev.example .env`.